src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -1,245 +1,249 @@
 'use strict';
 
-const angular = require('angular');
-
-const util = require('../../directive/test/util');
-
-describe('selectionTabs', function() {
-  const fakeSession = {
-    state: {
-      preferences: {
-        show_sidebar_tutorial: false,
-      },
-    },
-  };
-  const fakeSettings = {
-    enableExperimentalNewNoteButton: false,
+const { shallow, mount } = require('enzyme');
+const { createElement } = require('preact');
+
+const NewNoteBtn = require('../new-note-btn');
+const uiConstants = require('../../ui-constants');
+const SelectionTabs = require('../selection-tabs');
+
+describe('SelectionTabs', function() {
+  // mock services
+  let fakeSession;
+  let fakeSettings;
+
+  // default props
+  const defaultProps = {
+    isLoading: false,
+    isWaitingToAnchorAnnotations: false,
+    selectedTab: uiConstants.TAB_ANNOTATIONS,
+    totalAnnotations: 123,
+    totalNotes: 456,
+    totalOrphans: 0,
   };
 
-  before(function() {
-    angular
-      .module('app', [])
-      .component('selectionTabs', require('../selection-tabs'));
+  SelectionTabs.$imports.$mock({
+    '../store/use-store': callback =>
+      callback({
+        clearSelectedAnnotations: sinon.stub(),
+        selectTab: sinon.stub(),
+      }),
   });
 
-  beforeEach(function() {
-    const fakeStore = {};
-    const fakeFeatures = {
-      flagEnabled: sinon.stub().returns(true),
+  function createComponent(props) {
+    return shallow(
+      <SelectionTabs
+        session={fakeSession}
+        settings={fakeSettings}
+        {...defaultProps}
+        {...props}
+      />
+    ).dive();
+  }
+
+  // required for <Tab> rendering
+  function createDeepComponent(props) {
```

Comment:
An alternative to using `mount` here is to find the `Tab` component in the shallow-rendered output of `SelectionTabs` and then call `dive`:

```js
const tab = wrapper.find('Tab').first().dive();
assert.include(tab.text(), 'Annotations');
assert.include(tab.text(), '123');
```

This will avoid deep rendering all components (eg. `NewNoteBtn`).